cluster-autoscaler/processors/provreq/injector_test.go

```diff
@@ -117,7 +117,8 @@ func TestProvisioningRequestPodsInjector(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		client := provreqclient.NewFakeProvisioningRequestClient(context.Background(), t, tc.provReqs...)
-		injector := ProvisioningRequestPodsInjector{client, clock.NewFakePassiveClock(now)}
+		backoffTime := map[string]time.Duration{key(notProvisionedRecentlyProvReqB): 2 * time.Minute}
```

Comment:
Maybe add a new test case for backed off request scenario?

Reply:
I put backed off request scenario in a separate case